app/Main.hs

```diff
@@ -1,8 +1,43 @@
+{-# LANGUAGE OverloadedStrings #-}
+
 module Main where
 
+import Control.Concurrent.Async
+import qualified Control.Concurrent.Chan.Unagi as U
+import Control.Monad
+import Data.DateTime (getCurrentTime)
+import qualified Data.Text as Text
+import qualified Data.UUID as UUID
 import System.Environment (getEnv)
 
 import Lib
+import Registration
+import Mailer
 
 main :: IO ()
-main = getEnv "PORT" >>= someFunc . read
+-- main = getEnv "PORT" >>= someFunc . read
```

Comment:
Are you intending to return to this?

Reply:
Yes - that's the `Main` that was serving HTTP. I need to make one that goes back to doing that again. I really just wanted a temporary standalone exe that I could use to send real email in response to "requests".

Reply:
Gotcha.